concent_api/core/validation.py

```diff
@@ -411,3 +412,24 @@ def validate_secure_hash_algorithm(checksum: str):
             "Invalid SHA1 hash.",
             ErrorCode.MESSAGE_FILES_CHECKSUM_INVALID_SHA1_HASH
         )
+
+
+def validate_frames(frames_list: List[int]):
+    if not isinstance(frames_list, list) and len(frames_list) > 0:
```

Comment:
If `frames_list` will be for example `1`, this check will fail with `TypeError: object of type 'int' has no len()`. There should be `or` there instead of `and`.

Reply:
Done